lib/bloc/events/authentication_event.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:equatable/equatable.dart';
+import 'package:meta/meta.dart';
+
+@immutable
```

Comment:
why need this?

Reply:
because all properties in the inherited class is final and the instance of that class also final !